Test failures in CI

It looks like CI broke without any relevant changes in LibCST - my guess is that some change in pypi is responsible for the breakage.

The failures look like this:

```
 ======================================================================
FAIL: test_codemod_formatter_error_input (libcst.codemod.tests.test_codemod_cli.TestCodemodCLI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/LibCST/LibCST/libcst/codemod/tests/test_codemod_cli.py", line 37, in test_codemod_formatter_error_input
    self.assertIn(
AssertionError: 'error: cannot format -: Cannot parse: 13:10:     async with AsyncExitStack() as stack:' not found in 'Calculating full-repo metadata...\nExecuting codemod...\nFailed to determine module name for /home/runner/work/LibCST/LibCST/libcst/codemod/tests/codemod_formatter_error_input.py.txt: \'/home/runner/work/LibCST/LibCST/libcst/codemod/tests/codemod_formatter_error_input.py.txt\' is not in the subpath of \'/home/runner/work/LibCST/LibCST/libcst/tests/pyre\' OR one path is relative and the other is absolute.\nTraceback (most recent call last):\n  File "/opt/hostedtoolcache/Python/3.9.12/x64/bin/black", line 8, in <module>\n    sys.exit(patched_main())\n  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/black/__init__.py", line 1322, in patched_main\n    patch_click()\n  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/black/__init__.py", line 1308, in patch_click\n    from click import _unicodefun\nImportError: cannot import name \'_unicodefun\' from \'click\' (/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/site-packages/click/__init__.py)\n\r\x1b[2KCodemodding /home/runner/work/LibCST/LibCST/libcst/codemod/tests/codemod_formatter_error_input.py.txt\n\nTraceback (most recent call last):\n  File "/home/runner/work/LibCST/LibCST/libcst/codemod/_cli.py", line 311, in _execute_transform\n    newcode = invoke_formatter(config.formatter_args, newcode)\n  File "/home/runner/work/LibCST/LibCST/libcst/codemod/_cli.py", line 56, in invoke_formatter\n    subprocess.check_output(\n  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/subprocess.py", line 424, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,\n  File "/opt/hostedtoolcache/Python/3.9.12/x64/lib/python3.9/subprocess.py", line 528, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError: Command \'[\'/opt/hostedtoolcache/Python/3.9.12/x64/bin/black\', \'-\']\' returned non-zero exit status 1.\n\nFailed to codemod /home/runner/work/LibCST/LibCST/libcst/codemod/tests/codemod_formatter_error_input.py.txt\n\n\r\x1b[2K0.41s 0% complete, [calculating] estimated for 1 files to go...\r\x1b[2KFinished codemodding 1 files!\n - Transformed 0 files successfully.\n - Skipped 0 files.\n - Failed to codemod 1 files.\n - 0 warnings were generated.\n'
```
which looks like black failing under the hood.

--- a/libcst/_parser/parso/pgen2/grammar_parser.py
+++ b/libcst/_parser/parso/pgen2/grammar_parser.py
@@ -93,14 +93,10 @@ class GrammarParser:
     def _parse_items(self):
         # items: item+
         a, b = self._parse_item()
-        while (
-            self.type
-            in (
-                PythonTokenTypes.NAME,
-                PythonTokenTypes.STRING,
-            )
-            or self.value in ("(", "[")
-        ):
+        while self.type in (
+            PythonTokenTypes.NAME,
+            PythonTokenTypes.STRING,
+        ) or self.value in ("(", "["):
             c, d = self._parse_item()
             # Need to end on the next item.
             b.add_arc(c)

--- a/libcst/_parser/parso/python/token.py
+++ b/libcst/_parser/parso/python/token.py
@@ -26,7 +26,6 @@ try:
         ERRORTOKEN: TokenType = native_token_type.ERRORTOKEN
         ERROR_DEDENT: TokenType = native_token_type.ERROR_DEDENT
 
-
 except ImportError:
     from libcst._parser.parso.python.py_token import (  # noqa F401
         PythonTokenTypes,

--- a/libcst/_parser/parso/utils.py
+++ b/libcst/_parser/parso/utils.py
@@ -114,11 +114,11 @@ def python_bytes_to_unicode(
             return b"utf-8"
 
         # pyre-ignore Pyre can't see that Union[str, bytes] conforms to AnyStr.
-        first_two_match = re.match(br"(?:[^\n]*\n){0,2}", source)
+        first_two_match = re.match(rb"(?:[^\n]*\n){0,2}", source)
         if first_two_match is None:
             return encoding
         first_two_lines = first_two_match.group(0)
-        possible_encoding = re.search(br"coding[=:]\s*([-\w.]+)", first_two_lines)
+        possible_encoding = re.search(rb"coding[=:]\s*([-\w.]+)", first_two_lines)
         if possible_encoding:
             return possible_encoding.group(1)
         else:

--- a/libcst/codemod/commands/rename.py
+++ b/libcst/codemod/commands/rename.py
@@ -259,14 +259,11 @@ class RenameCommand(VisitorBasedCodemodCommand):
         inside_import_statement: bool = not self.get_metadata(
             QualifiedNameProvider, original_node, set()
         )
-        if (
-            QualifiedNameProvider.has_name(
-                self,
-                original_node,
-                self.old_name,
-            )
-            or (inside_import_statement and full_replacement_name == self.new_name)
-        ):
+        if QualifiedNameProvider.has_name(
+            self,
+            original_node,
+            self.old_name,
+        ) or (inside_import_statement and full_replacement_name == self.new_name):
             new_value, new_attr = self.new_module, self.new_mod_or_obj
             if not inside_import_statement:
                 self.scheduled_removals.add(original_node.value)

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,4 +1,4 @@
-black==21.10b0
+black==22.3.0
 coverage>=4.5.4
 fixit==0.1.1
 flake8>=3.7.8

